modules/upgrading-oadp10-to12-in-mtc.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/upgrading-3-4.adoc
+// * migration_toolkit_for_containers/upgrading-mtc.adoc
+
+:_content-type: PROCEDURE
+[id="migration-upgrading-oadp-for-mtc-18_{context}"]
+= Upgrading OADP 1.0 to 1.2 for {mtc-full} 1.8.0
+
+
+.Procedure
+
+
+* For each subscription, patch the OADP operator from OADP 1.0 to OADP 1.2, by issuing:
++
+[source,terminal]
+----
+$ oc -n openshift-migration patch subscription redhat-oadp-operator-stable-1.0-mtc-operator-catalog-openshift-marketplace --type merge --patch '{"spec": {"channel":"stable-1.2"}}'
+----
++
+[NOTE] 
+==== 
+Sections indicating the user-specific returned `NAME` values that are used for the installation of MTC & OADP, respectively. 
+While it will be similar, it is unlikely that the returned value will be `redhat-oadp-operator-stable-1.0-mtc-operator-catalog-openshift-marketplace`, which is used in this example.
+====
```

Comment:
While I see other updates, I'm not finding this in the rendered [preview](https://65453--docspreview.netlify.app/openshift-enterprise/latest/migration_toolkit_for_containers/upgrading-mtc#migration-upgrading-oadp-for-mtc-18_upgrading-mtc).  Is it something that I may need to wait longer because it's still building/refreshing or should it already be "live" in the preview?

Reply:
Looks ok to be - 
![image](https://github.com/openshift/openshift-docs/assets/106804821/fa3173d3-ed06-42e4-819f-37b24b15b4aa)

Reply:
It's there today when I look.  :man_shrugging: